File: app/src/main/java/com/example/myphotoapp/DB/DataBases.java
package com.example.myphotoapp.DB;

import android.provider.BaseColumns;

// DataBase Table
public final class DataBases {

    public static final class CreateDB implements BaseColumns {
        public static final String TITLE = "title";
        public static final String CONTENT = "content";
        public static final String PHOTO = "photo";
        public static final String _TABLENAME = "Memo";
        public static final String _CREATE =
                "create table "+_TABLENAME+"("
                        +_ID+" integer primary key autoincrement, "
                        +TITLE+" text not null , "
                        +CONTENT+" text not null , "
                        +PHOTO+" blob );";
    }
}

File: app/src/main/java/com/example/myphotoapp/DB/DB/DataBases.kt
package com.example.myphotoapp.DB.DB

import android.provider.BaseColumns

// DataBase Table
class DataBases {
    object CreateDB : BaseColumns {
        const val TITLE = "title"
        const val CONTENT = "content"
        const val PHOTO = "photo"
        const val _TABLENAME = "Memo"
        const val _CREATE = ("create table " + _TABLENAME + "("
                + BaseColumns._ID + " integer primary key autoincrement, "
                + TITLE + " text not null , "
                + CONTENT + " text not null , "
                + PHOTO + " blob );")
    }
}